bigcode_eval/tasks/humaneval.py

```diff
@@ -47,7 +47,16 @@ class GeneralHumanEval(Task):
 
     DATASET_PATH = "openai_humaneval"
 
-    def __init__(self, strip_prompt, k=[1, 10, 100], num_workers=16, timeout=3.0):
+    def __init__(
+            self, 
+            strip_prompt, 
+            k=[1, 10, 100], 
+            num_workers=16, 
+            timeout=3.0, 
+            humaneval_one_shot=False, 
```

Comment:
To avoid making this function too huge, we should can create a python dataclass nugget_config object